query-languages/m/m-spec-functions.md

```diff
@@ -86,18 +86,18 @@ An _argument-list_ is used to specify a fixed number of arguments directly as a
 
 The following holds when invoking a function:
 
-* The environment used to evaluate the _function-body_ of the function includes a variable that corresponds to each parameter, with the same name as the parameter. The value of each parameter corresponds to a value constructed from the _argument-list_ of the _invokeexpression_, as defined in [Parameters](#parameters). 
+* The environment used to evaluate the _function-body_ of the function includes a variable that corresponds to each parameter, with the same name as the parameter. The value of each parameter corresponds to a value constructed from the _argument-list_ of the _invoke-expression_, as defined in [Parameters](#parameters). 
 * All of the expressions corresponding to the function arguments are evaluated before the _function-body_ is evaluated.
 
-* Errors raised when evaluating the expressions in the _expression-list_ or _functionexpression_ are propagated.
+* Errors raised when evaluating the expressions in the _expression-list_ or _function-body_ are propagated.
```

Comment:
I think your change is correct, but it's slightly confusing whether the original author was intending the function-body or primary-expression. Given that it's listed after expression-list, your interpretation makes the most sense.

Reply:
Do you think it would make better grammatical sense to change this to the following?
> Errors raised when evaluating the expressions in the _expression-list_, or when evaluating the _function-body_, are propagated.

Reply:
No, I think what you have is fine.